apps/condo/domains/common/utils/select.utils.ts

```diff
@@ -0,0 +1,10 @@
+
+export function isNeedToLoadNewElements (scrollEvent, lastElementId: string, isLoading: boolean) {
```

Comment:
what about `checkIfScrollChanged`?

Reply:
no, it's not just check if scroll changed. This function returns true if we scrolled to the last element